pkg/controller/controller_actions_test.go

```diff
@@ -167,3 +168,23 @@ func checkCatalogClientActions(actual []testing.Action, expected []catalogClient
 	}
 	return nil
 }
+
+type brokerClientAction struct {
+	actionType fakeosb.ActionType
+}
+
+// checkBrokerClientActions is the utility function for checking actions returned by
+// the broker client
+func checkBrokerClientActions(actual []fakeosb.Action, expected []brokerClientAction) error {
```

Comment:
I'm a little worried that we have a few different ways of doing this type of check proliferating - did you see `assertGetCatalog` in `controller_test.go`?